### DISCUSSION TITLE
NES ROM Dumping - CHR and PRG settings lowest is 32?

### DISCUSSION THREAD

**Discussion by muguemjee** [sanni/cartreader#273] [Need help]
Im trying to dump a nes cart. It has chr and prg as 128 meaning on the sanni the values for those would be 16? Unless im mistaken and it should be literally the values?

If it should be 16 is there a problem with my device because 32 is the smallest value they can be set to?

In the video the sizes can be as small as 0. I see it goes like 0, 8, 16, 32, etc. On my device the lowest it goes is 32. Goes up to 512 then cycles back to 32

Ok so its the literal size values not divided by 8. But how come on my device it doesn't have 0, 8, 16 as value options?

**Comment by sanni** [comment]
The available options change with the selected mapper.
